classes/class.ilObjAdobeConnectGUI.php

```diff
@@ -2980,7 +2985,8 @@ public function editParticipants()
 		$my_tpl = new ilTemplate($this->pluginObj->getDirectory()."/templates/default/tpl.meeting_participant_table.html", true, true);
 
 		$has_access = ilXAVCPermissions::hasAccess($ilUser->getId(), $this->object->getRefId(), AdobeConnectPermissions::PERM_ADD_PARTICIPANTS);
-		if(count($this->object->getParticipantsObject()->getParticipants()) == 0 && $has_access)
+		$is_owner = ilObject::_lookupOwner($ilUser->getId()) == $ilUser->getId() ? true: false;
```

Comment:
`$is_owner = ilObject::_lookupOwner($ilUser->getId()) == $ilUser->getId();` should be sufficient.